Cargo.lock

```diff
@@ -2135,7 +2135,20 @@ checksum = "7baab56125e25686df467fe470785512329883aab42696d661247aca2a2896e4"
 dependencies = [
  "console",
  "lazy_static",
- "number_prefix",
+ "number_prefix 0.3.0",
+ "regex",
+]
+
+[[package]]
+name = "indicatif"
```

Comment:
suggestion(blocking): Regardless of whether we use or not use indicatiff, we shouldn't be adding a second copy of it. See indicatiff 0.13 above. So, we should either use 0.13 here and then send a follow-up to update  both cases, or we should first send an update PR for another usage, and then rebase this work on top.